homeassistant/components/neato/camera.py

```diff
@@ -60,18 +60,20 @@ def camera_image(self):
     def update(self):
         """Check the contents of the map list."""
         if self.neato is None:
-            _LOGGER.error("Error while updating camera")
+            _LOGGER.error("Error while updating '%s'", self._robot_name)
```

Comment:
Should we use entity ID instead ?

Reply:
Well there is only one platform entity per device and the default name is unique for the device. I think the name should help the user determine which device it is.

Reply:
Yup entity ID is much better, code is now updated.  Thanks for your suggestion!